### DISCUSSION TITLE
initial issues running test cases

### DISCUSSION THREAD

**Discussion by barlometer** [CH-Earth/summa#515] [Q&A] [answered]
Hi, 

I'm a beginner user.

I'm having issues running the summa test cases on Docker. 

After installing Docker, the container for summa, and test cases, I'm having issues running the test cases. 

This is my output when running ./runTestCases_docker.sh 

file_suffix is '_testSumma_docker'.
file_master is '/summaTestCases_2.x/settings/syntheticTestCases/celia1990/summa_fileManager_celia1990.txt'.
   1 SUMMA_FILE_MANAGER_V1.0: !

FATAL ERROR: summa_initialize/summa_SetTimesDirsAndFiles/unknown control file option: SUMMA_FILE_MANAGER_V1.0

STOP 1

Thanks in advance.

**Comment by wknoben** [comment, marked as answer]
@barlometer

Hi Nick,

Since this is (likely) not an issue with the SUMMA source code I'll close this and transfer it to the `Discussions` section. I'm reasonably certain I have an answer and easy fix for you.

**Reply by barlometer** [reply]
Ok, thanks.

**Reply by wknoben** [reply]
Following up, I think the issue may be that you're using the test cases configured for SUMMA v2 combined with the source code of SUMMA v3 (which is what you'll currently find on this repo's `master` and `develop` branches). We had a change in how the SUMMA configuration files are read between v2 and v3, and the old test cases are not functional anymore.

You can grab the new test cases from this (temporary) link: https://drive.google.com/file/d/1naiHcD_CuW88j9nvj8NAW6GDSMv29cOX/view

Let me know if this fixes the problem.

Cheers,
Wouter

**Reply by barlometer** [reply]
Ok, thanks for getting back to me. I'll give it a shot.

**Reply by wknoben** [reply]
No problem, let me know if it works or if you run into further issues.

**Reply by wknoben** [reply]
I'm going to mark this as answered, but feel free to get in touch again if needed.
